index.html

```diff
@@ -85,6 +86,176 @@ <h1>Hey There. I’m Akhtar</h1>
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </div>
+        <div class="card">
+            <h2 class="card-header">Profesional Art Printing Data</h2>
+            <p class="card-body">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the industry's standard</p>
+            <ul>
+                <li class="lang">
+                    <a href="#">CSS</a>
+                </li>
+                <li class="lang">
+                    <a href="#">bootstrap</a>
+                </li>
+                <li class="lang">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </div>
+        <div class="card">
+            <h2 class="card-header">Profesional Art Printing Data</h2>
+            <p class="card-body">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the industry's standard</p>
+            <ul>
+                <li class="lang">
+                    <a href="#">CSS</a>
+                </li>
+                <li class="lang">
+                    <a href="#">bootstrap</a>
+                </li>
+                <li class="lang">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </div>
+        <div class="card">
+            <h2 class="card-header">Profesional Art Printing Data</h2>
+            <p class="card-body">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the industry's standard</p>
+            <ul>
+                <li class="lang">
+                    <a href="#">CSS</a>
+                </li>
+                <li class="lang">
+                    <a href="#">bootstrap</a>
+                </li>
+                <li class="lang">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </div>
+        <div class="card">
+            <h2 class="card-header">Profesional Art Printing Data</h2>
+            <p class="card-body">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the industry's standard</p>
+            <ul>
+                <li class="lang">
+                    <a href="#">CSS</a>
+                </li>
+                <li class="lang">
+                    <a href="#">bootstrap</a>
+                </li>
+                <li class="lang">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </div>
+    </section>
+    <div class="about-me-main">

```

Comment:
- [ ] **Please make sure that the main container for about-myself section uses a `section` tag**

Section tag defines the section of documents. It is a semantic tag. There are various benefits to using semantic HTML. It helps SEOs, browsers and users to understand more about your content. In simple terms, it helps in better accessibility and search engine optimization.

Reply:
Thank you for the review. Section tag has been used this time in the ABOUT ME section

Reply:
:heavy_check_mark: